x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -297,6 +305,10 @@ function getBaseTemplate(
     data_stream: {
       timestamp_field: '@timestamp',
     },
+    _meta: {
+      package_name: packageName,
```

Comment:
Nit: You picked `package_name` instead of the initial `package.name`. The reason we made it an object is that we could potentially also add `package.version` and others in the future.